charts/external-dns/values.yaml

```diff
@@ -88,7 +88,7 @@ dnsPolicy:
 # -- [Init containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) to add to the `Pod` definition.
 initContainers: []
 
-# -- [Security context](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#securitycontext-v1-core) for the `external-dns` container.
+# -- [Pod security context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context) for the `external-dns` container.
```

Comment:
```suggestion
# -- [Security context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container) for the `external-dns` container.
```
Good spot on the broken link, I thought I'd got rid of them all. This is the security context for the ExternalDNS container not the pod security context which is `podSecurityContext` (which could also do with it's link updating to https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container).